eo-compiler/src/main/antlr4/org/eolang/compiler/Program.g4

```diff
@@ -199,5 +277,40 @@ NUMBER: ( '0' .. '9' )+;
 LETTER: ( 'a' .. 'z' | 'A' .. 'Z' | '0' .. '9' );
 TEXT: '"' ('\\"' | ~'"')* '"';
 
-SPACE: ' ';
-EOL: '\n';
+fragment SPACES: [ \t]+;
+
+NEWLINE
+ : ( {atStartOfInput()}?   SPACES
+   | ( '\r'? '\n' | '\r' ) SPACES?
+   )
+   {
```

Comment:
@g4s8 I realize that you got this code somewhere, right? In that case, we have to put a link to the original source.